src/utils/dateUtils.js

```diff
@@ -70,6 +70,9 @@ const dateUtils = {
         }
 
         if (_.isNumber(data)) {
+            /**
+             * L = 09/04/1986
```

Comment:
Might be better to say: `L = MM/DD/YYYY for enUS Locale.`
In some parts of the world 09/04/1986 => April 9, 1986 not September 4, 1986! 😁

Reply:
Fixed.

Reply:
Looks good.  Resolving.